### PR TITLE
PAYARA-2326 Added switch in glassfish-web to switch JAX-RS @RolesAllowed

### PR DESCRIPTION
**Pull request by arjantijms**
Signed-off-by: arjantijms <arjan.tijms@gmail.com>

### CODE REVIEW

**Comment by arjantijms** [comment]
Jenkins, quick test Payara 5 please

**Comment by arjantijms** [comment]
Jenkins, quick test Payara 5 please

**Comment by arjantijms** [comment]
Jenkins, quick test Payara 5 please

**Comment by arjantijms** [comment]
Jenkins, quick test Payara 5 please

File: appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/Application.java [hunk 1]
```diff
@@ -1757,7 +1762,7 @@ public void addResourceAdapter(String raName){
      * @return the Set of application paramaters.
      */
     public Set<ApplicationParam> getApplicationParams() {
-        return applicationParams; 
+        return applicationParams;
     }
 
    /**
```
Comment (smillidge) [inline appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/Application.java]: This file just seems to have a load of white space changes only is that correct?

File: appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/Application.java [hunk 2]
```diff
@@ -1782,15 +1787,15 @@ public boolean getKeepState() {
...
     public boolean getKeepStateResolved() {
         return keepStateResolved;
     }
 
     /**
-     * Sets the resolved keepstate value.  
+     * Sets the resolved keepstate value.
      * @param keepStateResolved
      */
     public void setKeepStateResolved(String keepStateResolved) {
```
Comment (smillidge) [inline appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/Application.java]: Again just white space changes is that correct?

**Pull request merged**